valintalaskenta-laskenta-service/src/main/java/fi/vm/sade/valintalaskenta/runner/util/AuthorizationUtil.java

```diff
@@ -0,0 +1,24 @@
+package fi.vm.sade.valintalaskenta.runner.util;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+/** TODO maybe move to java-cas or some other suitable place on a sunnier day. */
```

Comment:
On ollut kyl melko harmaata, olen nimittäin nähnyt tämän saman kommentin jossain muualla. Ota vaan todo pois.